src/akkd/Accordion/AccordionTitle.tsx

```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames'
+import React from 'react'
+import styles from './Accordion.module.css'
+
+interface Props {
+	index: any,
+	className?: string,
+	onClick?: (event: any, props: any) => void,
+}
+
+export class AccordionTitle extends React.PureComponent<Props> {
+	render() {
+		const {index, className, onClick = () => {}, children} = this.props
+		return (
+			<summary
+				className={classNames(
+					styles.accordionTitle,
+					className,
+				)}
+				onClick={(e)=>onClick(e, {index})}

```

Comment:
nit; avoid inline function calls in jsx
for class components, can break it out as a private arrow function in the class body, fn components can use a `React.useCallback`